### DISCUSSION TITLE
Today found self readme don't work

### DISCUSSION THREAD

**Discussion by xxl4** [community/community#76601] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Today, i found our github user profile don't work fine.
![image](https://github.com/community/community/assets/34465153/88da37f8-fe04-4cee-9f97-1e8074a3cd13)

when i check the code and use preview is ok

![image](https://github.com/community/community/assets/34465153/6735db92-d695-47bc-bf64-86ee188a11d9)

Please check it. 

Thank you.

**Comment by theaminulai** [comment]
it's well

**Comment by lotirium** [comment]
Don't worry it's working!
